### DISCUSSION TITLE
android 2.1.0

### DISCUSSION THREAD

**Discussion by lsnnt** [termux/termux-app#2776] [General]
if you have an apk file that can run in android 2.1.0 jellybean please mention it below.
i know that is is very old but i have root access in that mobile only and not in newer phones.

**Comment by lsnnt** [comment]
if you have any alternative you can also tell that here

**Comment by agnostic-apollo** [comment]
Termux only ever supported android 5+

**Comment by ghost** [comment]
There's no Android 2.1 Jellybean but gingerbread. but there's no such revision so prob you mean 4.1.0, You can use apps like Linux Deploy to run linux on that phone or use a jackpal's terminal emulator and run your binaries there

**Comment by ChiefMikeK** [comment]
If you're just looking for misc. old android stuff then try  
- https://archive.org/details/software?query=android
